### ISSUE TITLE
Select card not working on iPhone

### ISSUE THREAD

**Issue by bkr1969** [piitaya/lovelace-mushroom#975]
### Requirements

- [X] I have updated Mushroom to the latest available version
- [X] I did a search to see if there is a similar issue or if a pull request is open.

### Current Behavior

Mushroom-select-card shows as "Custom element doesn't exist..." on iPhone 13 (iOS 16.2)

### Expected Behavior

To work as it does on desktop

### Context

YAML state
```yaml

```

### Environment

```markdown
- Browser: HA App on iOS 16.2
- HA Version: 2023.1.7
- Mushroom: 2.5.2
```

**Comment by m1ckyb** [comment]
Happening to me as well, works fine in Edge & Firefox browsers i use, but on chrome (windows) and the home assistant app (android), getting Custom element doesn't exist...

**Comment by piitaya** [comment]
It's a cache/ressource loading issue. There is lot of closed issue for this problem. Close the app, restart HA server etc... It should be ok.

**Comment by bkr1969** [comment]
That doesn't fix it.  I don't generally run the app unless I'm looking for something in particular, so it has been force closed and restarted dozens of times. I've updated HA and made several edits throughout so I've re-started HA several times as well.

**Comment by bkr1969** [comment]
So this bug is specific to the HA app, not iPhone.  If I open my dashboard through Safari via NabuCasa, it works fine, but not with the app.

**Comment by townsmcp** [comment]
I agree. Terminating the app and restart HA server doesn’t fix the issue. 
for clarity, I am running HA 2023.2.0 and Companion App 2023.443 (was also happening in previous version of the app) on iOS 16.2

**Comment by piitaya** [comment]
My wife and I are using last iOS version on iPhone 13 and iPhone 11 and we don't have any issue. Did you try deleting mushroom resources (https://my.home-assistant.io/redirect/lovelace_resources/) and reinstalling Mushroom.

**Comment by townsmcp** [comment]
@piitaya im happy to give that a try. Would it kill all cards etc already setup though? Or will it all be fine after the reinstall?

Also, just to be clear, the Companion app version I have installed is from TestFlight rather than the AppStore

**Comment by townsmcp** [comment]
@piitaya thank you - that suggestion worked - Companion App now also shows the Select entity after following your suggested steps :)

> My wife and I are using last iOS version on iPhone 13 and iPhone 11 and we don't have any issue. Did you try this :
> 
> * delete mushroom resources (https://my.home-assistant.io/redirect/lovelace_resources/)
> * uninstall Mushroom from HACS
> * reinstall Mushroom from HACS

**Comment by bkr1969** [comment]
Yes, that fixed it for me as well.  Thanks!

**Comment by piitaya** [comment]
Cool so I will close the issue 🙂
